python/ray/scripts/scripts.py

```diff
@@ -938,6 +939,8 @@ def start(
     assert ray_params.gcs_address is not None
     ray._private.utils.write_ray_address(ray_params.gcs_address, temp_dir)
 
+    logger.info("Started Ray in: " + str(datetime.now() - start_time) + " seconds.")
```

Comment:
We should remove this before we merge it. Or we can add the second information around the starting log that we already have

Reply:
Moved it to the starting log.

Reply:
I don't see the change yet?

Reply:
sorry, pushed after responding, should be there now